### PR TITLE
update reference sn-react-typescript-boilerplate

### PR DESCRIPTION
**Pull request by sumirart**
update sn-react-typescript-boilerplate reference

### CODE REVIEW

File: _docs/tutorials/how-to-deploy-to-netlify.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ There are two ways you can host your site.
 
 ## Git Repository Setup
 
-We recommend you to use our GitHub template that you can find here [sn-react-typescript-boilerplate](https://github.com/SenseNet/sn-react-typescript-boilerplate). _If you don't know how to use a template repository check this [article](https://help.github.com/en/articles/creating-a-repository-from-a-template)_.
+We recommend you to use our GitHub template that you can find here [sn-react-typescript-boilerplate](https://github.com/SenseNet/sn-client/tree/master/examples/sn-react-typescript-boilerplate). _If you don't know how to use a template repository check this [article](https://help.github.com/en/articles/creating-a-repository-from-a-template)_.
```
Comment (zoltanbedi) [inline _docs/tutorials/how-to-deploy-to-netlify.md]:
```suggestion
We recommend you to use our starter template that you can find here [sn-react-typescript-boilerplate](https://github.com/SenseNet/sn-client/tree/master/examples/sn-react-typescript-boilerplate).
```

File: _docs/tutorials/getting-started-react.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ First of all you need a sensenet instance. You can create your own from GitHub o
 
 ## Use our starter
 
-To make things easier, we created a boilerplate app. It contains an example Hello World React SPA with sensenet repository login, written in Typescript. It is a template repository that you can find here [sn-react-typescript-boilerplate](https://github.com/SenseNet/sn-react-typescript-boilerplate). _If you don't know how to use a template repository check this [article](https://help.github.com/en/articles/creating-a-repository-from-a-template)_.
+To make things easier, we created a boilerplate app. It contains an example Hello World React SPA with sensenet repository login, written in Typescript. It is a template repository that you can find here [sn-react-typescript-boilerplate](https://github.com/SenseNet/sn-client/tree/master/examples/sn-react-typescript-boilerplate). _If you don't know how to use a template repository check this [article](https://help.github.com/en/articles/creating-a-repository-from-a-template)_.
```
Comment (zoltanbedi) [inline _docs/tutorials/getting-started-react.md]:
```suggestion
To make things easier, we created a boilerplate app. It contains an example Hello World React SPA with sensenet repository login, written in Typescript. You can find here [sn-react-typescript-boilerplate](https://github.com/SenseNet/sn-client/tree/master/examples/sn-react-typescript-boilerplate).
```

**Review by zoltanbedi — CHANGES_REQUESTED** [review]
This will no longer work as a template so we should reference it as a starter instead.

File: _docs/tutorials/how-to-deploy-to-netlify.md (hunk 1 above)
Comment (sumirart) [reply _docs/tutorials/how-to-deploy-to-netlify.md]: fixed on 0682e100dc19cdb1fdbb633e6eca45d8e76c16e5

thanks

File: _docs/tutorials/getting-started-react.md (hunk 2 above)
Comment (sumirart) [reply _docs/tutorials/getting-started-react.md]: fixed on 0682e100dc19cdb1fdbb633e6eca45d8e76c16e5

thanks

Comment (zoltanbedi) [reply _docs/tutorials/getting-started-react.md]: Can you also remove the last sentence from each paragraph? Thank you :)

Comment (sumirart) [reply _docs/tutorials/getting-started-react.md]: Sorry I didn't see it. Fixed on cffa5c0

Thanks

**Review by zoltanbedi — APPROVED** [review]
Awsome. Thanks 😀

**Pull request merged**